tox.ini

```diff
@@ -24,3 +18,4 @@ deps =
     django17: Django>=1.7,<1.8
     django18: Django>=1.8,<1.9
```

Comment:
I'm wondering why Django 1.9 is installed though it's not defined under the dependencies. Probably the `Django` package is pulled without a version restriction.

Can you add a `django19` line, please? Thank you.